<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterNewCard</class>
 <widget class="QDialog" name="RegisterNewCard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>572</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>注册银行卡</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pics/buttonIcon/buttonicon.qrc">
    <normaloff>:/Bank_202px_1194726_easyicon.net.png</normaloff>:/Bank_202px_1194726_easyicon.net.png</iconset>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>160</y>
     <width>571</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="bankGrid">
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
   </layout>
  </widget>
  <widget class="QLabel" name="selectedBankIcon">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>50</y>
     <width>200</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>持卡人姓名</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ownerName">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>70</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>211</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>持卡人身份证号</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>260</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>持卡人手机号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ownerPhone">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>260</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>六位支付密码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>450</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>重复密码</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="payCode">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>370</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="payCode_2">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>450</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly|Qt::ImhHiddenText|Qt::ImhNoAutoUppercase|Qt::ImhNoPredictiveText|Qt::ImhSensitiveData</set>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>80</y>
     <width>245</width>
     <height>63</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>这张卡所属的银行是</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="selectedBankName">
      <property name="font">
       <font>
        <family>Adobe Arabic</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="checkIcon_ownerID">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorText_ownerID">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>200</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入正确的18位身份证号！</string>
   </property>
  </widget>
  <widget class="QLabel" name="checkIcon_ownerName">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorText_ownerName">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入正确的姓名！</string>
   </property>
  </widget>
  <widget class="QLabel" name="checkIcon_ownerPhone">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorText_ownerPhone">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>300</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入正确的11位手机号！</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ownerID">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="checkIcon_payCode">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>410</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorText_payCode">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>410</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入正确的6位数字密码！</string>
   </property>
  </widget>
  <widget class="QLabel" name="checkIcon_payCode_2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>490</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="errorText_payCode_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>490</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>密码不一致！</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptBtn">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>460</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rejectBtn">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>460</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Adobe Arabic</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <zorder>gridLayoutWidget</zorder>
  <zorder>selectedBankIcon</zorder>
  <zorder>label_2</zorder>
  <zorder>ownerName</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>ownerPhone</zorder>
  <zorder>label_5</zorder>
  <zorder>label_6</zorder>
  <zorder>payCode</zorder>
  <zorder>payCode_2</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>ownerID</zorder>
  <zorder>checkIcon_ownerID</zorder>
  <zorder>errorText_ownerID</zorder>
  <zorder>checkIcon_ownerName</zorder>
  <zorder>errorText_ownerName</zorder>
  <zorder>checkIcon_ownerPhone</zorder>
  <zorder>errorText_ownerPhone</zorder>
  <zorder>checkIcon_payCode</zorder>
  <zorder>errorText_payCode</zorder>
  <zorder>checkIcon_payCode_2</zorder>
  <zorder>errorText_payCode_2</zorder>
  <zorder>acceptBtn</zorder>
  <zorder>rejectBtn</zorder>
 </widget>
 <tabstops>
  <tabstop>ownerName</tabstop>
  <tabstop>ownerID</tabstop>
  <tabstop>ownerPhone</tabstop>
  <tabstop>payCode</tabstop>
  <tabstop>payCode_2</tabstop>
 </tabstops>
 <resources>
  <include location="pics/buttonIcon/buttonicon.qrc"/>
 </resources>
 <connections/>
</ui>
